.gitignore

```diff
@@ -42,7 +42,7 @@ node_modules/
 jspm_packages/
 
 # Configuration file
-src/config.ts
+config/default.json
```

Comment:
Shouldn't the default.json exist by default?
The default flow should work without any modifications to the repo. And if the user wants to change the default mechanism, we will have to add in another json, and pass that to the config library (through env or any other mechanism)

Reply:
Makes sense, will update the PR